### PR TITLE
Feat : Profile

### PR DESCRIPTION
**Pull request by theejkb**
> Display the address, herotag, the nonce and shard of the connected wallet.

> Display every NFT on the wallet.

> Redirect to home when logged out.

### Visual preview :
navbar
profil

### CODE REVIEW

**Comment by theejkb** [comment]
navbar
profil

File: components/LoginModalButton.tsx [hunk 1]
```diff
@@ -37,6 +37,12 @@ export const LoginModalButton: FC<LoginModalButtonProps> = ({
     onClose: close,
   } = useDisclosure({ onClose, onOpen });
 
+  //Redirect to home when logged out
+  const handleLogout = () => {
+    logout();
+    window.location.href = '/';
```
Comment (juliancwirko) [inline components/LoginModalButton.tsx:43]: this should probably use next router

File: components/User.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { HomeSectionTitle } from './HomeSectionTitle';
+
+export const User = () => {
+  const [herotag, setHerotag] = useState('');
+  const [nonce, setNonce] = useState('');
+  const [shard, setShard] = useState('');
+
+  async function getHerotag() {
+    const res = await fetch(
+      `https://api.elrond.com/accounts/${accountState.address}`
```
Comment (juliancwirko) [inline components/User.tsx:13]: The API can't be hardcoded. There are env variables for that also, there are helpers for apiCalls in utils, and the dapp also uses SWR, which could be used here

File: components/User.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    getHerotag();
+  }, []);
+
+  return (
+    <Box>
+      <WrapItem justifyContent="center" alignItems="center">
+        <Avatar
+          size="xl"
+          name="NaaQ"
+          src="https://res.cloudinary.com/naaq/image/upload/v1653986673/t%C3%A9l%C3%A9chargement_aahft1.jpg"
```
Comment (juliancwirko) [inline components/User.tsx:32]: Hardcoded avatar?

File: components/UserNft.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import { useEffect, useState } from 'react';
+import { accountState } from '../store/auth';
+import { HomeSectionTitle } from './HomeSectionTitle';
+
+export const UserNft = () => {
+  const [nftCollections, setNftCollections] = useState([]);
+
+  async function getNftCollection() {
+    const res = await fetch(
+      `https://api.elrond.com/accounts/${accountState.address}/nfts`
```
Comment (juliancwirko) [inline components/UserNft.tsx:11]: the same as above, hardcoded api and we can use swr here

**Review by juliancwirko — CHANGES_REQUESTED** [review]
You can remove the package-lock.json from the PR. If merged, I will update all that is required.

File: components/User.tsx (hunk 3 above)
Comment (theejkb) [reply components/User.tsx:32]: I didn't found a way to get the profil image from Maiar App.

**Comment by theejkb** [comment]
> You can remove the package-lock.json from the PR. If merged, I will update all that is required.

Done ✅

File: components/User.tsx (hunk 3 above)
Comment (juliancwirko) [reply components/User.tsx:32]: yes, so I think we should remove the functionality of avatar

Comment (theejkb) [reply components/User.tsx:32]: Yep good idea

**Comment by juliancwirko** [comment]
It will be implemented here: https://github.com/ElvenTools/elven-tools-dapp/pull/20
I will close this one.

**Pull request closed without merging**